internal/storage/sql_provider.go

```diff
@@ -6,19 +6,31 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
+	"github.com/authelia/authelia/internal/logging"
 	"github.com/authelia/authelia/internal/models"
+	"github.com/authelia/authelia/internal/utils"
 )
 
 // SQLProvider is a storage provider persisting data in a SQL database.
 type SQLProvider struct {
-	db *sql.DB
+	db   *sql.DB
+	log  *logrus.Logger
+	name string
+
+	sqlUpgradesCreateTableStatements        map[int]map[string]string
+	sqlUpgradesCreateTableIndexesStatements map[int][]string
 
-	sqlCreateUserPreferencesTable            string
-	sqlCreateIdentityVerificationTokensTable string
-	sqlCreateTOTPSecretsTable                string
-	sqlCreateU2FDeviceHandlesTable           string
-	sqlCreateAuthenticationLogsTable         string
-	sqlCreateAuthenticationLogsUserTimeIndex string
+	/*
```

Comment:
can you remove this comment block?

Reply:
Yeah was just missed in my cleanup due to only committing part of my changes (individual lines). Fixed locally.